pkg/config/config.go

```diff
@@ -32,6 +34,19 @@ type Logging struct {
 	Console bool
 }
 
+type Certs struct {
+	CaPath   string
+	CertPath string
+}
+
+type Options struct {
+	PagedRpmInsertsLimit int
+}
+
+const (
+	DefaultPagedRpmInsertsLimit = 200
```

Comment:
i feel like 200 may be a tad low, but i did a few tries at 200 & 1000 and the time difference was minimal (3m54s vs 3m49s), so i'm okay with 200